projects/igniteui-angular/src/lib/buttonGroup/buttonGroup.component.ts

```diff
@@ -1,17 +1,24 @@
 import { CommonModule } from '@angular/common';
 import {
     AfterViewInit,
-    ChangeDetectorRef,
     Component,
-    ElementRef,
+    ContentChildren,
+    ChangeDetectorRef,
     EventEmitter,
-    forwardRef,
-    HostBinding, Inject,
-    Input, NgModule, Output, QueryList, Renderer2, ViewChildren
+    HostBinding,
+    Input,
+    NgModule,
+    Output,
+    QueryList,
+    Renderer2,
+    ViewChildren,
+    OnDestroy
 } from '@angular/core';
+import { Subject } from 'rxjs';
 import { IgxButtonDirective, IgxButtonModule } from '../directives/button/button.directive';
 import { IgxRippleModule } from '../directives/ripple/ripple.directive';
 import { IgxIconModule } from '../icon/index';
+import { takeUntil } from '../../../../../node_modules/rxjs/operators';
```

Comment:
Wrong import path. I guess it is VS Code automatic import resolution but this should just be `import .. from 'rxjs/operators`